packages/patternfly-4/react-core/src/components/Tabs/__snapshots__/Tabs.test.tsx.snap

```diff
@@ -1,1665 +1,1162 @@
-        id="tab2"
-        title="Tab item 2"
-      >
-        Tab 2 section
-      </ForwardRef>
-    }
-    id="tab2"
-    key="1/.1"
-  />
-  <ForwardRef
-    activeKey={0}
-    child={
-      <ForwardRef
-        eventKey={2}
-        id="tab3"
-        title="Tab item 3"
-      >
-        Tab 3 section
-      </ForwardRef>
-    }
-    id="tab3"
-    key="2/.2"
-  />
-</Fragment>
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
 `;
 
 exports[`should render tabs with eventKey Strings 1`] = `
-<Fragment>
-  <div
-    className="pf-c-tabs"
-    onSelect={[Function]}
-  >
-    <button
-      aria-label="Scroll left"
-      className="pf-c-tabs__scroll-button"
-      onClick={[Function]}
-    >
-      <AngleLeftIcon
-        color="currentColor"
-        noVerticalAlign={false}
-        size="sm"
-        title={null}
-      />
-    </button>
-    <ul
-      className="pf-c-tabs__list"
-      onScroll={[Function]}
-    >
-      <li
-        className="pf-c-tabs__item"
-        key="0/.0"
-      >
-        <ForwardRef
-          aria-controls="pf-tab-section-one-tab1"
-          className="pf-c-tabs__button"
-          eventKey="one"
-          id="tab1"
-          onClick={[Function]}
-          title="Tab item 1"
-        >
-          Tab item 1
-        </ForwardRef>
-      </li>
-      <li
-        className="pf-c-tabs__item"
-        key="1/.1"
-      >
-        <ForwardRef
-          aria-controls="pf-tab-section-two-tab2"
-          className="pf-c-tabs__button"
-          eventKey="two"
-          id="tab2"
-          onClick={[Function]}
-          title="Tab item 2"
-        >
-          Tab item 2
-        </ForwardRef>
-      </li>
-      <li
-        className="pf-c-tabs__item"
-        key="2/.2"
-      >
-        <ForwardRef
-          aria-controls="pf-tab-section-three-tab3"
-          className="pf-c-tabs__button"
-          eventKey="three"
-          id="tab3"
-          onClick={[Function]}
-          title="Tab item 3"
-        >
-          Tab item 3
-        </ForwardRef>
-      </li>
-    </ul>
-    <button
-      aria-label="Scroll right"
-      className="pf-c-tabs__scroll-button"
-      onClick={[Function]}
-    >
-      <AngleRightIcon
-        color="currentColor"
-        noVerticalAlign={false}
-        size="sm"
-        title={null}
-      />
-    </button>
-  </div>
-  <ForwardRef
-    activeKey={0}
-    child={
-      <ForwardRef
-        eventKey="one"
-        id="tab1"
-        title="Tab item 1"
-      >
-        Tab 1 section
-      </ForwardRef>
-    }
-    id="tab1"
-    key="0/.0"
-  />
-  <ForwardRef
-    activeKey={0}
-    child={
-      <ForwardRef
-        eventKey="two"
-        id="tab2"
-        title="Tab item 2"
-      >
-        Tab 2 section
-      </ForwardRef>
-    }
-    id="tab2"
-    key="1/.1"
-  />
-  <ForwardRef
-    activeKey={0}
-    child={
-      <ForwardRef
-        eventKey="three"
-        id="tab3"
-        title="Tab item 3"
-      >
-        Tab 3 section
-     
```

Comment:
@redallen first off, thanks for help. But are you sure this snap will actually cover test scenario? It's basically empty, and it doesn't look like it will ensure "should render tabs with eventKey Strings".
Appologies if it's a dumb question, i'm pretty new to jest testing, trying to understand to not introduce same mistakes in future PRs.

Reply:
The diff is too big so GH points to wrong line. My question was re this snap:
```
exports[`should render tabs with eventKey Strings 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
```

Reply:
Yep, it makes sense to make the snapshot tests that use OUIA use `mount` instead of `shallow` since there's now a `<ContextConsumer>`.